src/openMVG/features/image_describer.cpp

```diff
@@ -8,7 +8,9 @@
 #include "image_describer.hpp"
 
 #include "openMVG/features/image_describer_akaze.hpp"
+#if OPENMVG_IS_DEFINED(OPENMVG_HAVE_CCTAG)
```

Comment:
in theory you should need to include
```c++
#include <openMVG/config.hpp>
```
before using the #if. It may have been included by other #include but it's better to include-what-you-see